add todo steering committee blog post (#270)
todogroup/todogroup.org: content/en/blog/todo-steering-committee-2022.md

---
title: "Meet the TODO Steering Committee in 2022"
author: todogroup
date: 2021-12-27
---

> TODO Steering Committee is represented by global Open Source leaders and OSPO professionals, coming from a diverse range of industries.

Reviewer: ```suggestion
TODO Steering Committee is composed of global Open Source leaders and OSPO professionals from a diverse range of industries.
```

> Now that elections closed and results are published, we are proud to introduce the new Steering Committee members for 2022:

Reviewer: ```suggestion
Now that the 2021 election has been completed, we are proud to introduce the new Steering Committee members for 2022:
```

* [Ashley Wolf](https://twitter.com/Meta_Ashley) - GitHub
* [Dawn Foster](https://twitter.com/geekygirldawn) - VMware
* [Shilla Saebi](https://twitter.com/ShillaSaebi) - Comcast

> * [Stephen Augustus](https://twitter.com/stephenaugustus) - Cisco

Reviewer: ```suggestion
* [Stephen Augustus](https://whois.auggie.dev/) - Cisco
```

* [Stephen Jacobs](https://www.linkedin.com/in/itprofjacobs/) - RIT
* [Thomas Steenbergen](https://twitter.com/tsteenbe) - HERE Technologies
* [VM (Vicky) Brasseur](https://twitter.com/vmbrasseur) - Wipro

# Meet the Steering Commitee

***

Here is a short bio from each of the 2022 TODO Steering Committee representatives.

## Ashley Wolf - GitHub
**Remaining Member**

> Ashley is a passionate advocate for open source. She is currently the Open Source Program Manager at GitHub. Prior to GitHub, she led the Yahoo (acquired by Verizon) open source program and worked in product management for a cybersecurity company. Ashley serves on steering committees for open source foundations and is an advisor for Built by Girls, an organization that prepares the next generation of female and non-binary leaders to step boldly into careers powered by technology. 

Reviewer: ```suggestion
Ashley is a passionate advocate for open source. She is currently the Open Source Program Manager at GitHub. Prior to GitHub, she led the Yahoo (acquired by Verizon) open source program and worked in product management for a cybersecurity company. Ashley serves on steering committees for open source foundations and is an advisor for Built by Girls, an organization that prepares the next generation of female and non-binary leaders to step boldly into careers powered by technology.
```

Most recently, Dawn was a consultant at The Scale Factory after being Director of Community at Puppet. Prior to Puppet, she was leading the Community Office within Intel’s Open Source Technology Center. In addition to working at Intel, Dawn was an online community consultant, and she has worked at Jive Software, Compiere, and a Midwestern manufacturing company in positions ranging from Unix system administrator to market researcher to community manager to open source strategist.

Dawn holds a PhD from the University of Greenwich, an MBA from Ashland University, and a BS in Computer Science from Kent State University. Dawn blogs about online communities as the author of the Fast Wonder Blog, and she's blogged for The New Stack, Linux.com, GigaOM’s WebWorkerDaily, and in various other places. She is the author of the books, What Dawn Eats: Vegan Food That Isn’t Weird and Companies and Communities: Participating without being sleazy.

## Shilla Saebi - Comcast

> **New Member**

Reviewer: ```suggestion

**New Member**
```

From 2014-2017, she was nominated to serve as a member of the OpenStack User Committee by the existing board members, becoming the first woman to serve on OpenStack's UC. She was also a core contributor to the OpenStack docs project and currently sits on the Editorial Advisory Board for OpenStack Superuser Magazine. Shilla also served on the board of the OpenStack Innovation Center (OSIC) from 2015-2017, where she helped the community develop and test code at scale. She formed the OpenStack Northern Virginia meetup group which has over 650 members and is currently an OpenStack ambassador for the East Coast of the United States.

Most recently, Shilla started the Philadelphia Prometheus meetup group, and is helping with a series of internal Kubernetes training workshops at Comcast. She has authored several blog posts on open source technology and has spoken about cloud operations, diversity, and open source community building at many technology conferences around the globe

## Stephen Augustus - Cisco

> **New Member**
> 

Reviewer: ```suggestion

**New Member**
```

Stephen is a Black engineering director and leader in open source communities. He is Cisco’s first Head of Open Source, within the Emerging Technologies & Incubation division.

For Kubernetes, he has co-founded transformational elements of the project, including the KEP (Kubernetes Enhancements Proposal) process, the Release Engineering subproject, and Working Group Naming. Stephen has also previously served as a chair for both SIG PM and SIG Azure. He continues his work in Kubernetes as a Steering Committee member and a Chair for SIG Release.

> Across the wider CNCF (Cloud Native Computing Foundation) ecosystem, Stephen has the pleasure of serving as a SIG Contributor Strategy Chair, and a maintainer for the Dex project. Previously, he was one of the Program Chairs for KubeCon / CloudNativeCon, the cloud native community’s flagship conference. He is a prolific contributor to CNCF projects, amongst the top 25 (as of writing) code/content committers, all-time.

Reviewer: ```suggestion
Across the wider CNCF (Cloud Native Computing Foundation) ecosystem, Stephen has the pleasure of serving as a TAG Contributor Strategy Chair, and a maintainer for the Dex project. Previously, he was one of the Program Chairs for KubeCon / CloudNativeCon, the cloud native community’s flagship conference. He is a prolific contributor to CNCF projects, amongst the top 25 (as of writing) code/content committers, all-time.
```

In 2020, Stephen co-founded the Inclusive Naming Initiative, a cross-industry group dedicated to helping projects and companies make consistent, responsible choices to remove harmful language across codebases, standards, and documentation. He leads the Community/Open Source workstream and maintains the initiative’s infrastructure.

## Stephen Jacobs - RIT University

> **Remaining Member**

Reviewer: ```suggestion

**Returning Member**
```

Stephen Jacobs is a Professor at the Rochester Institute of Technology and Director of Open@RIT, the school's Open Programs Office. He led the design of their academic minor in Free and Open Source Software and Free Culture. His efforts in teaching FOSS courses began with an Honors Seminar in which students created educational games for the One Laptop per Child XO. His research activities in FOSS have been supported by the Ford Foundation, the Alfred P. Sloan Foundation, Red Hat, Inc and AT&T Inc.

## Thomas Steenbergen - HERE Technologies

> **New Member**
> 

Reviewer: ```suggestion

**New Member**
```

Thomas Steenbergen is the Head of Open Source at HERE Technologies with a background in various software technologies. He works with his core and extended teams to improve using and contributing to Open Source projects, and building open source communities. Thomas is also an active open source contributor for many years. He has been contributing to the SPDX specification for software bill of materials since 2015, helping to make it better match what developers find in code. Thomas work on open-sourced review tooling solutions within ClearlyDefined and OSS Review Toolkit projects, and also build wider open source communities as one of the co-founders of the European Chapter of the TODO group and the Open source Tooling Group.

Thomas is a frequent speaker and panellist at various global open source conferences and am always happy to start a conversation around anything open source. I’ve held a variety of technical lead roles over the past 15 years, and have worked in Netherlands, UK and Germany.

## Vicky Brasseur - Wipro

> **New Member**
> 

Reviewer: ```suggestion

**New Member**
```

VM (aka Vicky) spent most of her twenty-plus years in the tech industry leading software development departments and teams, providing corporate strategy, technical management, and leadership consulting for businesses, helping companies understand, use, release, and contribute to free and open source software in a way that’s good for both their bottom line and for the community.

She is the author of Forge Your Future with Open Source, the first and only book to detail how to contribute to free and open source software projects. The book is published by The Pragmatic Programmers and is available at https://fossforge.com.

Vicky has been a moderator and author for opensource.com, an author for Linux Journal, the Vice President of the Open Source Initiative, and is a frequent and popular speaker at free/open source conferences and events. She’s the proud winner of the Perl White Camel Award (2014), the O’Reilly Open Source Award (2016), and two Opensource.com Moderator’s Choice Awards (2018, 2019). She blogs about free/open source, business, and technical management at {anonymous => ‘hash’}

> 
> 

Reviewer: ```suggestion
```

# Big Thanks

> 
> *** 

Reviewer: ```suggestion
```

> Last but not least, the TODO Group would like to extend a big 'thanks' to the outgoing TODO Members: Chris Aniszczyk , Duane O'Brien, Stormy Peters and Kevin P. Fleming.  Thanks to their guidance and expertise in OSPOs, they made outstanding contributions to the project governance and the community.

Reviewer: ```suggestion
Last but not least, the TODO Group would like to extend a big 'thanks' to the outgoing Steering Committee members: Chris Aniszczyk, Duane O'Brien, Stormy Peters, and Kevin P. Fleming. Thanks to their guidance and expertise in OSPOs, they have made outstanding contributions to the project's governance and the community.
```

> If you wish to learn more about the Steering Commitee please check [TODO Governance repo](https://github.com/todogroup/governance).

Reviewer: ```suggestion
If you want to learn more about the Steering Committee, please check out the [TODO Governance repo](https://github.com/todogroup/governance).
```
